src/game/engine/foot.h

```diff
@@ -167,12 +169,12 @@ class FootClass : public TechnoClass
     target_t m_SuspendedNavCom;
     target_t m_NavList[NAV_LENGTH];
     GamePtr<TeamClass> m_Team;
-    TeamNumberType m_field_113;
-    FootClass *m_field_114; // Next team member?
+    TeamGroupType m_Group;
+    FootClass *m_Member; // Next linked team member.
     FacingType m_Paths[PATH_LENGTH];
-    MoveType m_PathMove; // The move type the current path was generated with.
+    MoveType m_PathBrk; // The move type the current path was generated with. // Perhaps "PathBreak"?
```

Comment:
I'd prefer PathBreak, PathBrk seems unneccessarily concise? I'm aware its what the symbols show it was called originally.